composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=8.0.0"
```

Comment:
Interesting. This would allow to tag 1.1.. =)

Just saying.. I know you have a plan.